### ISSUE TITLE
Update CONTRIBUTING because `docker-compose` CLI is deprecated

### ISSUE THREAD

**Issue by embano1** [awslabs/eventbridge-kafka-connector#463]
https://github.com/awslabs/eventbridge-kafka-connector/blob/9ccaba83b3db9943c334fa8170e5d7ec02873f55/CONTRIBUTING.md?plain=1#L95

**Comment by embano1** [comment]
```
WARN[0000] /Users/micha/dev/eventbridge-kafka-connector/e2e/docker_compose.yaml: the attribute `version` is obsolete, it will be ignored, please remove it to avoid potential confusion

```

**Comment by agebhar1** [comment]
https://docs.docker.com/compose/install/linux/#install-the-plugin-manually

**Comment by embano1** [comment]
Yeah, I was wondering if we need this because it ships by default with docker - but good point that I'll leave a hint that if `docker compose` doesn't work, the plugin can be installed manually.

**Comment by agebhar1** [comment]
On my Linux system, the package does not contain `docker-compose` by default.

**Comment by embano1** [comment]
Do you have Docker Desktop installed? If so, it should support `docker compose` (note the missing `-`). If you don't have Docker Desktop installed, then yes, manual instructions are required.

**Comment by agebhar1** [comment]
No, I have the plain/normal Docker package installed.
